build-edk2/Dockerfile

```diff
@@ -17,6 +17,11 @@ RUN wget -q -O gcc-riscv.tar.gz https://github.com/riscv-collab/riscv-gnu-toolch
 RUN echo "c3b849d25a43d74e7eb1cf739a8a9fadba5f997c4814732650f36ae250a8ab00 gcc-riscv.tar.gz" | sha256sum --check
 RUN tar -C /opt -xzf gcc-riscv.tar.gz
 
+# Use the LoongArch64 cross toolchain for building LoongArch64 code. 
+RUN wget https://github.com/loongson/build-tools/releases/download/2023.08.08/x86_64-cross-tools-loongarch64-binutils_2.41-gcc_13.2.0.tar.xz
```

Comment:
Let's check the downloaded file's hash, similar to this: https://github.com/rust-osdev/ovmf-prebuilt/blob/c836bb328e35093e506037e553902add4a59efc0/build-edk2/Dockerfile#L17